.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: c
+dist: trusty
+before_script:
+- wget -qO- http://coda.cs.cmu.edu/coda/jaharkes.key | sudo apt-key add -
+- echo -e "deb http://coda.cs.cmu.edu/coda trusty main\ndeb-src http://coda.cs.cmu.edu/coda trusty main" | sudo tee /etc/apt/sources.list.d/coda.list
+- sudo apt-get update
+- sudo apt-get build-dep --yes coda
+script:
+- ./bootstrap.sh && ./configure && make -j16 && make -j16 check && sudo make install
```

Comment:
Although travis examples all put the configure/make in the 'script' section, when a script step fails it continues with the next, while in before_script it will lead to a build failure. So it seems to me that the bootstrap/configure/make steps should be in before_script, while make check/make distcheck should be in script.